UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Changeling/ChangelingAbility/Implementation/AbsorbAbility.cs

```diff
@@ -29,16 +29,15 @@ public override bool UseAbilityServer(ChangelingMain changeling, Vector3 clickPo
 			Chat.AddCombatMsgToChat(changeling.gameObject,
 			$"<color=red>You start absorbing of {target.visibleName}</color>",
 			$"<color=red>{changeling.ChangelingMind.CurrentPlayScript.visibleName} starts absorbing of {target.visibleName}</color>");
-			var cor = changeling.StartCoroutine(AbsorbingProgress(stingTime / 10f, target));
+			Chat.AddExamineMsg(target.gameObject, "<color=red>Your body is absorbing!</color>");
```

Comment:
```suggestion
			Chat.AddExamineMsg(target.gameObject, "<color=red>Your body is being absorbed!</color>");
```

Do we have any reference from SS13? Maybe the text their used was better.